proxy/api/src/main.rs

```diff
@@ -20,46 +32,121 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let mut args = pico_args::Arguments::from_env();
     let args = Args {
-        reset_state: args.contains("--reset-state"),
```

Comment:
Now that this is removed, we can close: https://github.com/radicle-dev/radicle-upstream/issues/960.
However we also need to update the QA docs: https://github.com/radicle-dev/radicle-upstream/blob/master/QA.md#prerequisites.

@kim Would you prefer to do that in this issue or should we open another one?

Reply:
If the PR affects those bits we should address it in here imo